package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node app.js",
-    "setdb":"node sync-db.js"
+    "start": "npx nodemon app.js",
```

Comment:
just use nodemon app.js instead of npx nodemon

Reply:
> just use nodemon app.js instead of npx nodemon

Hey, I have modified it. Thanks for pointing it out.